### ISSUE TITLE
Wanted: search command

### ISSUE THREAD

**Issue by artemave** [facebook/PathPicker#129]
Search-based navigation is faster than pressing j n times. It would be great to have `/` command with vim like behaviour.

Thanks for a great tool anyway!

**Comment by pcottle** [comment]
Thanks @artemave ! This came up internally at Facebook and there were a number of interesting questions:

-- Should search only match file paths or should it match against the whole input?
-- Would it be case sensitive? Auto-case sensitive? support regexes?
-- Would we allow specifying a fuzzy regex to filter the input first before displaying it?

And then once we started exploring this idea, it became clear that PathPicker would start to try to do "too many things" at once, and the more `unixy` way would be to preprocess the input before sending it to PathPicker. So we decided this was a non-goal.

However something like `fzf` would be a great tool to use in this case:
https://github.com/junegunn/fzf

Closing out since we won't realistically get to this, but if you have ideas for how to make fpp play better with certain workflows im interested!
